linode/acceptance/tmpl/e2e_test_firewall.gotf

```diff
@@ -22,7 +19,7 @@ locals {
 }
 
 resource "linode_firewall" "e2e_test_firewall" {
-    depends_on = local.firewall_dependencies
+    depends_on = [ local.ipv4_address ]
```

Comment:
Did you try remove this `depends_on`? I feel it might work because TF might recognize `local.ipv4_address` and `local.ipv6_address` usage below and automatically build up the dependency relationship, though I didn't test it.

Reply:
I added it for safety but I can remove it as tests seem to be passing regardless